src/puppetbridge.ts

```diff
@@ -513,7 +558,12 @@ export class PuppetBridge extends EventEmitter {
 
 		// check if this is a valid room at all
 		const room = await this.namespaceHandler.createRoom(roomData);
-		if (!room || room.isDirect) {
+		if (room) {
+			this.metrics.room.inc({type: room.isDirect ? "dm" : "group", protocol: this.protocol?.id});
```

Comment:
This only increments for group-chats, never for DMs - @Sorunome am i checking the correct thing here? is `room.isDirect` guaranteed always set correctly here?